### ISSUE TITLE
Duplicates in THREAD_TABLE.scad

### ISSUE THREAD

**Issue by CrashCash** [adrianschlatter/threadlib#66]
Just wanted to mention a lot of the lines (924 to be exact) in THREAD_TABLE.scad are duplicates. This brings the line count from 2,825 to 1,901.

**Comment by adrianschlatter** [comment]
Indeed. I think that happens when long and short designators of the thread are identical. We should only generate the long descriptor when it is different from the short descriptor.
